src/zoid/buttons/component.jsx

```diff
@@ -451,6 +451,12 @@ export const getButtonsComponent : () => ButtonsComponent = memoize(() => {
                 value:      getMerchantID
             },
 
+            buttonsArray: {
+                type:       'array',
+                queryParam: true,
+                value:      ({ props }) => getButtonsArray(props)
```

Comment:
Let's update this name to match. Either `getButtons` or `getRenderedButtons`
```suggestion
                value:      ({ props }) => getRenderedButtons(props)
```

Reply:
Done!